File: python3/1234.py
while True:
    try:
        entrada = input()
        
        upper = True
        tam = len(entrada)
        saida = []

        for i in range(tam):
            if(entrada[i].isalpha()):
                if(upper):
                    saida += entrada[i].upper()
                    upper = False
                else:
                    saida += entrada[i].lower()
                    upper = True
            else:
                saida += entrada[i]

        for i in range(tam):
            print(saida[i], end="")
        print()

    except EOFError:
        break

File: cpp/strings/1234.cpp
#include <iostream>
#include <cstdio>

using namespace std;

int main() {

    //freopen("1234.in", "r", stdin);
    string entrada;

    while(getline(cin, entrada)) {

        bool upper = true;
        int tam = entrada.size();

        for(int i = 0; i < tam; i++) {
            if(isalpha(entrada[i])) {
                if(upper) {
                    entrada[i] = toupper(entrada[i]);
                    upper = false;
                } else {
                    entrada[i] = tolower(entrada[i]);
                    upper = true;
                }
            }
        }
        cout << entrada << endl;
    }

    return 0;
}

